src/environment.js

```diff
@@ -135,6 +135,13 @@ export default class Environment extends EventEmitter {
       this.dir = process.cwd();
     }
 
+    if (!this.dest) {
+      this.dest = 'sassdoc';
+    }
+
+    this.dest = this.resolve(this.dest, this.destCli);
+    this.destName = path.relative(process.cwd(), this.dest);
```

Comment:
I find the keys `destName` and `themeName` a bit confusing, since they are actually paths. (relative but still).